jdbi/src/main/java/org/killbill/commons/jdbi/guice/DataSourceProvider.java

```diff
@@ -200,7 +206,13 @@ DataSource buildDataSource() {
                 }
             }
 
-            return new HikariDataSource(hikariConfig);
+            try {
+                return new HikariDataSource(hikariConfig);
+            } catch (final PoolInitializationException e) {
+                // When initializationFailFast=true, log the exception to alert the user (the Guice initialization sequence will continue though)
```

Comment:
> Guice initialization sequence will continue though

Maybe something we should address-- i think leaving the stack up in that state, is not so great.